client2/src/game/Game.js

```diff
@@ -572,7 +572,7 @@ class Game extends Phaser.Scene {
             // Send player hitPoints and websocket to be used by chatInput
             // player is undefined 'cause it's not defined,
             PubSub.publish(ENTER_KEY, {
-                hitPoints: this.player.hitPoints,
+                hitPoints: PlayerState.hitPoints,
```

Comment:
Here's the fixed implementation that should resolve this [concern](https://github.com/Arcanorum/dungeonz/pull/99/commits/836dbbae4fa2d910c1b996d4ce08192f8eaef81f#r570315945)